src/components/Learningpath/LearningpathIframe.tsx

```diff
@@ -72,7 +72,7 @@ const LearningpathIframe = ({ html, url }: Props) => {
     const iframe = getIframeDOM();
     if (iframe) {
       const rect = iframe.getBoundingClientRect();
-      const scrollTop = window.pageYOffset || document.documentElement.scrollTop;
+      const scrollTop = window.scrollY || document.documentElement.scrollTop;
```

Comment:
Er dette en fiks for noe annet?

Reply:
![image](https://github.com/user-attachments/assets/48ce5de9-2cce-4cfb-b7e5-167f98f2a50d)